packages/nx/src/tasks-runner/cache.ts

```diff
@@ -181,6 +181,10 @@ export class Cache {
   }
 
   private copy(src: string, directory: string): Promise<void> {
+    // 'cp -a /path/dir/ dest/' operates differently to 'cp -a /path/dir dest/'
+    // path.resolve() removes trailing slashes
+    src = resolve(src);
```

Comment:
There are cheaper ways to remove trailing slash than `resolve`. Why not use `regex` with `replace`?

e.g. `src.replace(/\/$/, '')`

Reply:
I was unsure that assuming forward slashes would work in all circumstances in a platform independent manner. 

Thoughts?

Reply:
It would probably be best to check for either slash, but that can still be a regex:

```
src.replace(/[\/\\]$/, '')
```

Reply:
Sure, thanks. Will do.

Reply:
Thank you!